src/renderer/components/layout/page-layout.scss

```diff
@@ -42,6 +42,16 @@
       width: 218px;
       padding: 60px 10px 60px 20px;
 
+      h2 {
+        margin-bottom: 10px;
+        font-size: 18px;
+        padding: 6px 10px;
+        overflow-wrap: anywhere;
+        color: var(--textColorAccent);
+        text-transform: uppercase;
```

Comment:
I think that `capitalize` would be a better text-transform.

Reply:
You might be right, will test.

Reply:
Ended up to remove whole text-transform, does not make sense here.

Reply:
sounds good.